### PR TITLE
fix(RHTAPREL-842): FBC should error when preGA & hotfix both set

### PR DESCRIPTION
**Pull request by theflockers**
This commit adds a check to make sure fbc.preGA and fbc.hotfix can not be both set at once

Signed-off-by: Leandro Mendes <lmendes@redhat.com>

### CODE REVIEW

File: tasks/add-fbc-contribution/add-fbc-contribution.yaml [hunk 1]
```diff
@@ -98,6 +98,11 @@ spec:
         # default target_index
         target_index=$(params.targetIndex)
 
+        if [ "${hotfix}" == "true" ] && [ "${pre_ga}" == "true" ]; then
+          echo "fbc.preGA and fbc.hotfix are mutually exclusive. Please set just one in the releasePlanAdmission"
```
Comment (davidmogar) [inline tasks/add-fbc-contribution/add-fbc-contribution.yaml]:
```suggestion
          echo "fbc.preGA and fbc.hotfix are mutually exclusive. Please set just one in the ReleasePlanAdmission"
```

**Review by davidmogar — CHANGES_REQUESTED** [review]

File: tasks/add-fbc-contribution/add-fbc-contribution.yaml (hunk 1 above)
Comment (theflockers) [reply tasks/add-fbc-contribution/add-fbc-contribution.yaml]: fixed!

**Review by davidmogar — APPROVED** [review]

**Review by johnbieren — APPROVED** [review]

**Comment by jinqi7** [comment]
@theflockers , the e2e-tests of you PR will fail until the [PR](https://github.com/redhat-appstudio/e2e-tests/pull/1132) is merged.

**Pull request merged**